processor/resourcedetectionprocessor/README.md

```diff
@@ -17,6 +17,14 @@ details of which are currently pending confirmation in the OpenTelemetry specifi
     * host.name
     * os.type
 
+system detector custom configuration example:
```

Comment:
This should have the same indentation as the `* System metadata` bullet point above. Also
```suggestion
System detector custom configuration example:
```

Reply:
I have added the suggested changes.